docs/configuration.md

```diff
@@ -698,39 +698,6 @@ This storage backend uses Amazon's Simple Storage Service (S3).
   </tr>
 </table>
 
-### Maintenance
```

Comment:
@donhcd The top here might be better. It is hard to tell this applies to all storage drivers when wading through each specific configuration section.